### PR TITLE
1.9/1506 new eligibility

### PR DESCRIPTION
**Pull request by charlesstrange2**
Ok, so, we eventually settled on select boxes because of space requirements
- updated the formrequests and controllers to store eligibility.
- added a config variable for options
- rewrote seeds to use the config vars
- kick started the use of translation system
- made a migration for live-only to change the default eligibility

https://trello.com/c/Z3KLtDeJ/1506-new-way-of-recording-eligibility-vvv-ee

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,3 +18,4 @@ npm-debug.log
 /public/css
 /public/fonts
 /public/js
+/resources/fonts/dompdf_font_family_cache.php
```
Comment (katjam) [inline .gitignore]: Wasn't this in another PR also?

File: app/Http/Controllers/Store/RegistrationController.php [hunk 2]
```diff
@@ -448,14 +449,11 @@ function ($new_carer) {
             (array)$request->get('children')
         );
 
-        // Grab the date
-        $now = Carbon::now();
```
Comment (katjam) [inline app/Http/Controllers/Store/RegistrationController.php]: Assuming `$now` and `$user` were unused from before and this is a tidy up?

File: resources/views/store/create_registration.blade.php [hunk 3]
```diff
@@ -93,19 +93,20 @@
...
-                               @if(old('eligibility') == "other") checked="checked" @endif
-                        />
-                        <label for="other">Other Local Criteria</label>
-                    </div>
+                    <select name="eligibility" id="eligibility-reason">
+                        @foreach (config('arc.reg_eligibilities') as $index => $reg_eligibility)
+                            <option value="{{ $reg_eligibility }}"
+                                    @if(
+                                        (empty(old('eligibility')) && $index === 0) ||
+                                        old('eligibility') == "healthy-start-receiving"
```
Comment (katjam) [inline resources/views/store/create_registration.blade.php]: `===` for good luck?

File: resources/views/store/create_registration.blade.php [hunk 4]
```diff
@@ -93,19 +93,20 @@
...
+                                        ) selected="selected"
+                                    @endif
+                            >@lang('arc.reg_eligibilities.' . $reg_eligibility)
```
Comment (katjam) [inline resources/views/store/create_registration.blade.php]: Neat!

File: .gitignore (hunk 1 above)
Comment (charlesstrange2) [reply .gitignore]: yes,i suspect it got whacked by another branch.

File: app/Http/Controllers/Store/RegistrationController.php (hunk 2 above)
Comment (charlesstrange2) [reply app/Http/Controllers/Store/RegistrationController.php]: yep, IDE highlighted them as unused.

File: resources/views/store/create_registration.blade.php (hunk 3 above)
Comment (charlesstrange2) [reply resources/views/store/create_registration.blade.php]: ok :four_leaf_clover: will amend

**Pull request merged**